api/v2/views/instance_action.py

```diff
@@ -12,11 +12,11 @@ class InstanceActionFilter(filters.FilterSet):
     provider_id = django_filters.CharFilter(method='filter_by_provider')
     instance_id = django_filters.CharFilter(method='filter_by_instance')
 
-    def filter_by_instance(self, queryset, value):
+    def filter_by_instance(self, queryset, query_key, query_value):
```

Comment:
Also you changed from `value` to `query_value` here and below, but not in the previous places. Should they all be changed?